nuxeo-importer-queues/src/main/java/org/nuxeo/ecm/platform/importer/queue/manager/CQManager.java

```diff
@@ -15,43 +16,56 @@
  *     bdelbosc
  */
 
-
-import net.openhft.chronicle.queue.ChronicleQueue;
-import net.openhft.chronicle.queue.ExcerptAppender;
-import net.openhft.chronicle.queue.ExcerptTailer;
-import net.openhft.chronicle.queue.impl.single.SingleChronicleQueueBuilder;
-import org.jetbrains.annotations.Nullable;
-import org.nuxeo.common.utils.ExceptionUtils;
-import org.nuxeo.ecm.platform.importer.log.ImporterLogger;
-import org.nuxeo.ecm.platform.importer.source.SourceNode;
+import static org.apache.commons.io.FileUtils.deleteDirectory;
 
 import java.io.File;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
-import static org.apache.commons.io.FileUtils.deleteDirectory;
+import org.jetbrains.annotations.Nullable;
+import org.nuxeo.common.utils.ExceptionUtils;
+import org.nuxeo.ecm.platform.importer.log.ImporterLogger;
+import org.nuxeo.ecm.platform.importer.source.SourceNode;
+
+import net.openhft.chronicle.queue.ChronicleQueue;
+import net.openhft.chronicle.queue.ExcerptAppender;
+import net.openhft.chronicle.queue.ExcerptTailer;
+import net.openhft.chronicle.queue.impl.single.SingleChronicleQueueBuilder;
 
 /**
  * @since 8.10
  */
 public class CQManager extends AbstractQueuesManager {
 
-    final List<ChronicleQueue> queues;
-    final List<ExcerptAppender> appenders;
-    final List<ExcerptTailer> tailers;
+    List<ChronicleQueue> queues;
```

Comment:
Why drop the `final` on these? Also don't use package-private declarations, these should be `protected` (to allow use in subclasses).